nasa/ow_autonomy
# PLEXIL uses new Discard/Deliver actions

Before, PLEXIL used only the Deliver action, which was used for both discarding and delivering samples.  Now it calls Discard and Deliver separately as needed, and gives no arguments to Deliver.

To test:

1. Start the Atacama world.
2. Run the test plan: `roslaunch ow_plexil ow_exec.launch plan:=TestDiscardDeliver.plx`
3. See that the Discard and Deliver look correct, and that the plan runs without error.

NOTE: there are some minor and unrelated formatting and cosmetic edits in a few files.
